app/src/main/java/org/mozilla/fenix/settings/SecretDebugSettingsFragment.kt

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings
+
+import android.os.Bundle
+import androidx.preference.Preference
+import androidx.preference.PreferenceFragmentCompat
+import com.leanplum.Leanplum
+import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.showToolbar
+
+/**
+ * Lets the user customize Private browsing options.
```

Comment:
Could you add a bit more here? I don't think this description matches my understanding of what this settings menu is doing